kernel/src/table_features/mod.rs

```diff
@@ -492,7 +490,7 @@ static DELETION_VECTORS_INFO: FeatureInfo = FeatureInfo {
     feature_type: FeatureType::ReaderWriter,
     feature_requirements: &[],
     read_support: KernelSupport::Supported,
-    write_support: KernelSupport::NotSupported,
+    write_support: KernelSupport::Supported,
```

Comment:
Keep the comment close to the code :)
```suggestion
    /// We support writing to tables with DeletionVectors enabled, but we never write DV files
    /// ourselves (no DML). The kernel only performs append operations.
    write_support: KernelSupport::Supported,
```